.github/workflows/deploy.yml

```diff
@@ -0,0 +1,92 @@
+name: Trigger Target Workflow
+
+on:
+  workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      
+      - name: Upgrade pip
+        run: |
+          python -m pip install --upgrade pip
+      
+      - name: install reproschema
+        run: |
+          pip install -U reproschema==0.7.2
```

Comment:
do we always want to use the same version? what if we fix something later in the tool and have a new version? (maybe it's better to use the same version to be consistent

Reply:
I can remove the version, however, I kept the version there as, during testing, a weird bug where pip kept installing different versions. This line forced it to be the version with the converter implemented.

Reply:
sounds good to me